share/geojson/import.go

```diff
@@ -22,6 +22,37 @@ type GeoJSON struct {
 	Features []GeoFeature `json:"features"`
 }
 
+type GeoFeatureUS struct {
+	Type       string        `json:"type"`
+	Properties PropertiesUS  `json:"properties"`
+	Geometry   schema.Geometry      `json:"geometry"`
+}
+
+type Geometry struct {
+	Type        string      `bson:"type"`
+	Coordinates interface{} `bson:"coordinates"`
+}
+
+type Boundary struct {
+	Country  string   `bson:"country"`
+	State    string   `bson:"state"`
+	County   string   `bson:"county"`
+	Geometry Geometry `bson:"geometry"`
+}
+
+type PropertiesUS struct {
+	Intptlat   string    `json:"intptlat"`
+	GeoPoint2d []float64 `json:"geo_point_2d"`
+	Stusab     string    `json:"stusab"`
+	Namelsad   string    `json:"namelsad"`
+	Awater     int       `json:"awater"`
+}
+
+type GeoJSONUS struct {
```

Comment:
There is no bad to create a `GeoJSONUS ` since the property is varied by source. But I will suggest to re-use existed `GeoJSON` struct with a `map[string]interface{}` as properties.